routes/socket.js

```diff
@@ -60,11 +72,16 @@ const handleSocketConnection = async (io, socket) => {
   socket.on('task:answer', (answer) => {
     logger.info('task:answer', { socketMessage: answer, roomId, gameMasterSocketId });
     io.to(gameMasterSocketId).emit('task:answer', { answer, player: socket.id });
+    taskService.checkAnswer(answer, room, player);
   });
 
   socket.on('round:ending', (task) => {
     logger.info('round:ending', { socketMessage: task, roomId });
-    socket.to(`room-${roomId}`).emit('round:ending', { task });
+    room.getPlayers().then((players) => {
+      players.forEach((playerObj) => {
+        io.to(playerObj.socket).emit('round:ending', { task, score: playerObj.score });
```

Comment:
is there any way of sending the position compared to other players so I can display it on the end of round screen? Also, why is the task sent or am I missing something, since this is round ending?

Reply:
Round end is also task end, but at this time I'm not sure if we use it for any thing, shall add the position.

Reply:
I don't need the task at task end also since I am just sending the answers to you when the button is clicked and I don't do any validation at task end (as currently constructed), just thought to let you know